src/commands/context/reply-with-issue.ts

```diff
@@ -51,45 +49,62 @@ export const responses: Option[] = [
       'Replies with directions for questions with not enough information',
     reply: {
       title: 'Please add more information to your question',
-      content:
+      description:
         'Your question currently does not have sufficient information for people to be able to help. Please add more information to help us help you, for example: relevant code snippets, a reproduction repository, and/or more detailed error messages. See more info on how to ask a good question in https://discord.com/channels/752553802359505017/1138338531983491154 and https://discord.com/channels/752553802359505017/752553802359505020/1108132433917919292.',
     },
   },
+  {
+    name: 'Crossposting or Reposting',
+    description: 'Keep the question in one channel and wait for a response',
+    reply: {
+      title:
+        'Crossposting and reposting the same question across different channels is not allowed',
+      description:
+        'Crossposting (posting a question in a channel and send the question link to another channel) and reposting (posting the same question in several channels) are not allowed in this server. See the server rules in https://discord.com/channels/752553802359505017/752553802359505020/1108132432609284187 for more information.',
+    },
+  },
   {
     name: 'Improve Forum Question Title',
     description:
       'Tell the user to update their question title to make it more descriptive',
     reply: {
       title: 'Please improve the title of your question',
-      content:
+      description:
         'To ensure you get the best possible assistance, could you please change your thread title to be more descriptive? Specific titles attract the attention of users who can help and make it easier for others to find similar solutions in the future. You can change the title by going to `•••` → `Edit Post` → `Post Title`.',
     },
   },
   {
-    name: 'Crossposting or Reposting',
-    description: 'Keep the question in one channel and wait for a response',
+    name: 'Use Code Blocks',
     reply: {
-      title:
-        'Crossposting and reposting the same question across different channels is not allowed',
-      content:
-        'Crossposting (posting a question in a channel and send the question link to another channel) and reposting (posting the same question in several channels) are not allowed in this server. See the server rules in https://discord.com/channels/752553802359505017/752553802359505020/1108132432609284187 for more information.',
+      title: 'Please use code blocks',
+      description: [
+        'When sharing code or error messages, please use code blocks.',
+        'You can create a code block by wrapping your code in three backticks(`), like this: ```\ncode here\n```',
+        'You can also specify the language in the code block(like ts, js) to enable syntax highlighting, like this(It may not work for mobile):  ```ts\nexport default function Page(){}\n```',
```

Comment:
```suggestion
        'You can also specify the language in the code block (e.g. `ts`, `js`) to enable syntax highlighting:  ```ts\nexport default function Page(){}\n```',
```